crates/re_space_view_spatial/src/parts/spatial_scene_part_data.rs

```diff
@@ -1,14 +1,14 @@
-use crate::scene::UiLabel;
+use crate::parts::UiLabel;
 
 /// Common data struct for all spatial scene elements.
 ///
 /// Each spatial scene element is expected to fill an instance of this struct with its data.
-pub struct SpatialScenePartData {
+pub struct SpatialViewPartSystemData {
```

Comment:
I don't think we need System here.
"The SpatialViewPartSystem fills out an instance of the SpatialViewPartData" sounds right to me.